transit/transit_types.hpp

```diff
@@ -206,7 +206,7 @@ class Gate
   void SetBestPedestrianSegment(SingleMwmSegment const & s) { m_bestPedestrianSegment = s; };
 
   FeatureId GetFeatureId() const { return m_featureIdentifiers.GetFeatureId(); }
-  OsmId GetOsmId() const { return m_featureIdentifiers.GetOsmId(); }
+  OsmId GetSerialId() const { return m_featureIdentifiers.GetSerialId(); }
```

Comment:
I think `GetSerialId` is not a good name here.

Reply:
Right.
done